src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -6512,6 +6512,13 @@ static void M2()
             await VerifyNoItemsExistAsync(markup);
         }
 
+        [WorkItem(7788)]
```

Comment:
`[WorkItem(7648, "https://github.com/dotnet/roslyn/issues/7648")]` (7788 is the PR)